src/main/java/nl/praegus/fitnesse/responders/allTags/AllTagsResponder.java

```diff
@@ -0,0 +1,77 @@
+package nl.praegus.fitnesse.responders.allTags;
+
+import fitnesse.FitNesseContext;
+import fitnesse.authentication.SecureOperation;
+import fitnesse.authentication.SecureReadOperation;
+import fitnesse.authentication.SecureResponder;
+import fitnesse.http.Request;
+import fitnesse.http.Response;
+import fitnesse.http.SimpleResponse;
+import fitnesse.wiki.*;
+import fitnesse.wikitext.parser.SourcePage;
+import org.json.JSONObject;
+
+import java.io.UnsupportedEncodingException;
+import java.util.*;
+
+public class AllTagsResponder implements SecureResponder {
+    private JSONObject tagObject = new JSONObject();
+
+    @Override
+    public Response makeResponse(FitNesseContext fitNesseContext, Request request) throws Exception {
+        WikiSourcePage sourcePage = new WikiSourcePage(loadPage(fitNesseContext, request.getResource(), request.getMap()));
+
+        return makeTagResponse(sourcePage);
+    }
+
+    public SimpleResponse makeTagResponse(SourcePage sourcePage) throws UnsupportedEncodingException {
+        tagObject.put("Tags", getPageTags(sourcePage, new HashSet<>()));
+
+        SimpleResponse response = new SimpleResponse();
+        response.setMaxAge(0);
+        response.setStatus(200);
+        response.setContentType("application/json");
+        response.setContent(tagObject.toString(3));
+        return response;
+    }
+
+    public Set<String> getPageTags(SourcePage page, Set<String> allTags){
+        String[] tags = page.getProperty(WikiPageProperty.SUITES).split("\\s*,\\s*");
+        for(String tag: tags) {
```

Comment:
deze hele loop kan vervangen worden door een mooie stream volgens mij: (snap dat dit wat geavanceerd is :-)) 
allTags.addAll(Arrays.stream(tags).filter(t -> t.length()>0).collect(Collectors.toList()));

Reply:
Stream geimplementeerd. 

Moest dit wel even goed bestuderen, maar het is zeker handiger dan een for loop.